plugins/k8sauditeks/pkg/k8sauditeks/k8sauditeks.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package k8sauditeks
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"time"
+
+	"github.com/alecthomas/jsonschema"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk/plugins"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk/plugins/source"
+	"github.com/falcosecurity/plugin-sdk-go/pkg/sdk/symbols/extract"
+	"github.com/falcosecurity/plugins/plugins/k8saudit/pkg/k8saudit"
+	"github.com/falcosecurity/plugins/shared-aws/shared/go/aws/cloudwatchlogs"
+	"github.com/falcosecurity/plugins/shared-aws/shared/go/aws/session"
+)
+
+const pluginName = "k8sauditeks"
+
+type Plugin struct {
+	k8saudit.Plugin
+	Config PluginConfig
+}
+
+type PluginConfig struct {
+	Region          string `json:"region"           jsonschema:"description=The Region of your EKS cluster (default: us-east-1), it can be replaced by env var AWS_REGION"`
```

Comment:
Same as before: Region, Profile, and Cluster name are good open parameters candidates.